analytics/contentsquare/experience-accelerator/src/Client.ts

```diff
@@ -0,0 +1,94 @@
+export abstract class Client {
```

Comment:
We really need to extract out Client -- this version is slightly different, and it's going to be pretty error prone to join them back together soon